lms/djangoapps/course_blocks/utils.py

```diff
@@ -0,0 +1,32 @@
+"""
+Common utilities for use along with the course blocks.
+"""
+import json
+from courseware.models import StudentModule
+
+
+def get_student_module_as_dict(user, course_key, block_key):
```

Comment:
@andy-armstrong: I moved (and updated) this existing code to reuse it.  However, I didn't at a true unit test.  It is currently being tested via an integration unit test.  

I had a quick discussion with @nasthagiri about this.  Is this ok long-term?  Is this ok short-term for keeping down the cost of a refactor?  Do we just need to rely on code coverage to understand if we have an existing test?